backend/src/models/user.model.ts

```diff
@@ -0,0 +1,17 @@
+import { Schema, model } from 'mongoose';
+
+interface UserModel {
+    first_name: string,
+    last_name: string,
+    user_name: string,
+    persons: [Schema.Types.ObjectId]
+}
+
+const schema = new Schema<UserModel>({
+    first_name: { type: String, required: true },
+    last_name: { type: String, required: true },
+    user_name: { type: String, required: false },
+    persons: { type: [Schema.Types.ObjectId] }
```

Comment:
should there also be an array of `encounters` associated with this user's account for a bidirectional relationship?

Reply:
Yeah, that might be a good idea. It'll allow us to quickly fetch them all rather than going through the list of people first.